src/adapters/websocket-adapter.ts

```diff
@@ -13,14 +10,17 @@ interface EventListeners {
  * @param {EventListeners} eventListeners - An object with optional properties that will be called when certain event occurs
  * @example
  * const url = `${socketURL}`
- * const eventListeners = {
- *   typingStatus: (typing) => {
- *     console.log(`The user is ${typing ? 'typing' : 'not typing'}`)
- *   }
+ * const eventListeners{
+ *   typingStatus: typing => 
+ *   console.log(`The user is ${typing}`)
  * }
  * userTypingStatusAdapter(eventListeners)
  */
-export const userTypingStatusAdapter = async (eventListeners:EventListeners) => {
+export const userTypingStatusAdapter = async(eventListeners:EventListeners) => {
+
+    const jwtCookie = getCookie(cookieName)as { token: string }
+    const jwtToken = jwtCookie ? jwtCookie.token: null
+
     const socket: Socket = io(`${socketURL}/?token=${jwtToken}`)
 
     socket.on('connect' , () =>{
```

Comment:
@larasodjati I think we should call the event itself here which is typingStatus, and for connect you can write another eventListener . like it is in my code.